opencontrols/data.go

```diff
@@ -0,0 +1,28 @@
+package opencontrols
+
+import (
+	"github.com/opencontrol/compliance-masonry/commands/docs/docx"
```

Comment:
We shouldn't import anything from `commands` folder but we can't do anything right now lol